src/Accounts.vala

```diff
@@ -30,6 +31,15 @@ public class Tootle.Accounts : Object {
                 updated (saved_accounts);
             },
             network.on_show_error);
+
+        info ("Getting information from %s", accounts.formal.instance);
+        msg = new Soup.Message ("GET", "%s/api/v1/instance".printf (accounts.formal.instance));
+        network.inject (msg, Network.INJECT_TOKEN);
```

Comment:
Does this API call require auth?

Reply:
No, it's not needed , it's public
![imagen](https://user-images.githubusercontent.com/11057356/79995174-2dc52d80-84b7-11ea-9c03-aaf7e59d6c60.png)

Reply:
Can you remove injection here then? It's unneccessary and may confuse someone.

Reply:
Thank you!